src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -115,6 +115,18 @@ def try_path_munging(
     return result
 
 
+def set_tags(scope: Scope, result: JSONData) -> None:
```

Comment:
These are the tag setting that happens at the end of the iteration of the code mappings.